bootstrap/command.go

```diff
@@ -89,8 +110,13 @@ func Main(ctx *blueprint.Context, config interface{}, extraNinjaFileDeps ...stri
 		defer trace.Stop()
 	}
 
-	if flag.NArg() != 1 {
-		fatalf("no Blueprints file specified")
+	filesToParse, err := listFiles()
+	if err != nil {
+		if flag.NArg() != 1 {
+			fatalf("no Blueprints file specified, and "+
+				"could not read list file because: %v\n", err.Error())
+		}
+		filesToParse = []string{flag.Arg(0)}
```

Comment:
I suspect that we don't want to be passing both a list of files via listFiles() and also a root file via flag.Arg(0) . I'm still investigating a better way make that work

Reply:
Yeah, we might delete the root Android.bp file.  On the other hand, people will expect it to exist so maybe we will replace it with one containing a comment that explains the new system, in which case we can punt on supporting invoking blueprint without a root file until later.